### DISCUSSION TITLE
[question] How to remove -Wl,--no-undefined from shared library link line?

### DISCUSSION THREAD

**Discussion by yurivict** [mesonbuild/meson#10558] [Q&A] [answered]
```-Wl,--no-undefined``` seems to be always added.I need to remove it for a workaround for one unrelated issue.
How to remove ```-Wl,--no-undefined```?

Thanks,
Yuri

**Comment by eli-schwartz** [comment, marked as answer]
This is controlled by the `b_lundef` built-in option, which defaults to `true`, but you want to set to `false`.
